test/model/trivial/test_moving_average.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# First-party imports
+from gluonts.dataset.common import ListDataset
+from gluonts.model.trivial.mean import MovingAveragePredictor
+
+# Third-party imports
+import numpy as np
+import pytest
+
+
+def get_predictions(
+    target, prediction_length=1, context_length=1, freq="D", start="2020"
+):
+    mp = MovingAveragePredictor(
+        prediction_length=prediction_length,
+        context_length=context_length,
+        freq=freq,
+    )
+
+    ds = ListDataset([{"target": target, "start": start}], freq=freq)
+    item = next(iter(ds))
+    predictions = mp.predict_item(item).samples[0]
```

Comment:
```suggestion
    predictions = mp.predict_item(item).mean
```

I like this a bit better.